Backend/src/main/java/lt/vu/ads/models/Account.java

```diff
@@ -3,19 +3,30 @@
 import lombok.Getter;
 import lombok.Setter;
 
+import javax.persistence.*;
+
 @Getter
 @Setter
+@MappedSuperclass
 public class Account {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false)
```

Comment:
I am not sure, but if ID is auto generated, I think it won't be null.

Reply:
I used Intellij auto generated code, but it should not impact anything if i delete that @Column row. Will change